.cargo/config.toml

```diff
@@ -1,3 +1,52 @@
 [env]
 TG_ID = ""
 TG_HASH = ""
+
+# Build configuration
+[build]
+# Number of parallel jobs, defaults to # of CPUs
+# jobs = 8
+# Increase if you have sufficient RAM
+# Default is 16, increase for better performance with enough RAM
+incremental = true
+# Enable for faster rebuilds
+rustflags = ["-C", "link-arg=-fuse-ld=lld", "-C", "target-cpu=native"]
+
+# Target-specific settings
+[target.x86_64-pc-windows-msvc]
+linker = "lld-link"
+
+[target.x86_64-unknown-linux-gnu]
+linker = "clang"
+rustflags = ["-C", "link-arg=-fuse-ld=lld"]
+
+# Alias for common commands
+[alias]
+# Run tests with all features
+test-all = "test --all-features"

```

Comment:
if you introduce these beforehand, you can use them in CI. i also don't think these aliases are worth it.